model/Service/Properties/provider.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# provider
+
+## Summary
+
+The provider of a SoftwareService.
+
+## Description
+
+The Agent that provides a SoftwareService to one or more consumers.
+
+## Metadata
+
+- name: provider
+- Nature: ObjectProperty
+- Range: ServiceProvider
```

Comment:
is `ServiceProvider` already defined

Reply:
Yes - it is already in the service-profile branch (previous commit)

Reply:
Ah, I tried to grep for it and failed to find it.

Reply:
You're probably on the wrong branch - I just grepped the `service-profile` branch with the following results:

```
$ egrep -ri serviceprovider ./
./Service/Classes/ServiceProvider.md:# ServiceProvider
./Service/Classes/ServiceProvider.md:A ServiceProvider is an entity that provide a software service.
./Service/Classes/ServiceProvider.md:- name: ServiceProvider
```